versions/3.0.md

```diff
@@ -2355,7 +2355,12 @@ description: Pets operations
 
 #### <a name="examplesObject"></a>Examples Object
 
-Anywhere an `example` may be given, a JSON Reference MAY be used, with the 
+Instead of a single `example`, you may provide an array of 
+[Example Object](#exampleObject) values named `examples'.
+`example` and `examples` are mutually exclusive.
+
+Anywhere an `example` may be given (including in the Examples Object array), 
+a JSON Reference MAY be used, with the 
```

Comment:
For consistency, we should use `Reference Object` _(linkfied)_ instead of `JSON Reference`.